script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -391,67 +418,75 @@ That's so... sad.
 [sha:07eea14e3babd84451283f9273a391e9c0591340]{
 -- Page 412, Offset 34461.
 -- こんな時に、それが嘘だと分かってしまうのは、皮肉すぎる。
-At a time like this, I'm finally able to see through her lies. How ironic.
+Only now that things have come to this am I able to see through her lies...
+ The irony is crushing.
 }
 [sha:b7b35791f18e1a1a2752c305d04e05a1c88e995e]{
 -- Page 413, Offset 34462.
 -- 「……ひどいな。最後まで、先輩は俺に嘘をつくのか」
-"...How awful. Do you intend on lying to me until the very end, Senpai?"
+"I can't believe it...
+ Do you really plan on lying to me right up until the very end, Senpai?"
 }
 [sha:f0dd4a6a183c39fe5dad0df55f71db7a3d60ca20]{
 -- Page 413, Offset 34463.
 -- 「―――――――」
 "―――――――"
 }
-[sha:68a938c0eaed172db6b8953cba8d5faa7e319dae]{
+[offset:34464]{
 -- Page 413, Offset 34464.
 -- 返答はない。
-There's no answer.
+She doesn't answer.
 }
 [sha:f417c9d90a852cbaa37487186c9fa1669775b646]{
 -- Page 413, Offset 34465.
 -- 凍りついたように、先輩は動かない。
-Senpai is completely frozen in place.
+Senpai is motionless, as though frozen in place.
```

Comment:
Yeah I polled some people as well and the `as if`s take it, so who knows what went wrong with my upbringing, though for this _exact_ phrase people did like `as though` because it avoids the double fricative